projects/plugins/jetpack/extensions/blocks/dialogue/edit.js

```diff
@@ -216,8 +216,14 @@ export default function DialogueEdit( {
 						return createBlock( blockNameFallback );
 					}
 
+					// Copy attrs for the new block
+					// only if content is not empty.
+					const newAttributes = value?.length
+						? attributes
```

Comment:
Are there other attributes like the timestamp, which we'd want to persist even when the content is empty?

Reply:
yes, we'd like to copy the timestamp because it supposed two adjacents Dialogue blocks should have similar timestamp values. IN the end, it will help the user to sync the transcription with the media source. Even more, we could be smarter and increase or decrease the timestamp value.